include/seqan3/io/sequence_file/format_genbank.hpp

```diff
@@ -0,0 +1,266 @@
+                           {std::istreambuf_iterator<char>{stream},
+                            std::istreambuf_iterator<char>{}};
+        auto stream_it = std::ranges::begin(stream_view);
+
+        std::string idbuffer;
+        std::ranges::copy(stream_view | view::take_until_or_throw(is_cntrl || is_blank),
+                          std::back_inserter(idbuffer));
+
+        if (idbuffer != "LOCUS")
+            throw parse_error{"An entry has to start with the code word LOCUS."};
+
+        //ID
+        if constexpr (!detail::decays_to_ignore_v<id_type>)
+        {
+            if (options.embl_genbank_complete_header)
+            {
+                while (idbuffer != "ORIGIN")
+                {
+                        std::ranges::copy(idbuffer  | view::char_to<value_type_t<id_type>>,
+                                     	  std::back_inserter(id));
+                        std::ranges::copy(stream_view | view::take_until_or_throw(is_char<'\n'>)
+                                                	  | view::char_to<value_type_t<id_type>>,
+                                     	  std::back_inserter(id));
+                        std::ranges::copy(std::string_view{"\n"} | view::char_to<value_type_t<id_type>>,
+                                     	  std::back_inserter(id));
+
+                    ++stream_it;
+                    idbuffer.clear();
+                    std::ranges::copy(stream_view | view::take_until_or_throw(is_cntrl || is_blank),
+                                 	  std::back_inserter(idbuffer));
+                }
+            }
+            else
+            {
+				// ID
+                detail::consume(stream_view | view::take_until(!is_blank));
+                // read id
+                std::ranges::copy(stream_view | view::take_until_or_throw(is_cntrl || is_blank)
+                                              | view::char_to<value_type_t<id_type>>,
+                                  std::back_inserter(id));
+                detail::consume(stream_view | view::take_line_or_throw);
+            }
+        }
+
+        // Jump to sequence
+        while (idbuffer != "ORIGIN") //True, if not complete header is read into id
+        {
+            detail::consume(stream_view | view::take_until(is_cntrl));
+            ++stream_it;
+            idbuffer.clear();
+            std::ranges::copy(stream_view | view::take_until_or_throw(is_cntrl || is_blank),
+                              std::back_inserter(idbuffer));
+        }
+
+        // Sequence
+        detail::consume(stream_view | view::take_line_or_throw);
+        auto constexpr is_end = is_char<'/'> ;
+        if constexpr (!detail::decays_to_ignore_v<seq_type>)
+        {
+            auto seq_view = stream_view | ranges::view::remove_if(is_space || is_digit)
+            // ignore whitespace and numbers
+                                        | view::take_until_or_throw(is_end);   // until //
```

Comment:
```suggestion
                                        | view::take_until_or_throw_and_consume(is_end);   // until '//'
```
If you use `and_consume` then `//` is automatically consumed and you do not need to skip it at the end

Also, since you do not use seq_view anywhere, you can put the views direclty in the copy statement.